### ISSUE TITLE
Empty signup screen on iOS simulator/device

### ISSUE THREAD

**Issue by vijayendra-tripathi** [aws-amplify/aws-sdk-ios#1613]
I have configured AWS Cognito using Amplify. Please review following screenshot.
![SignIn](https://user-images.githubusercontent.com/11576681/59630673-8e7f2d80-9163-11e9-9bb8-7a3ef1bb06fb.png)

Following is screenshot of my configuration details.
![Cognito-Screenshot](https://user-images.githubusercontent.com/11576681/59630718-aa82cf00-9163-11e9-9e29-51f85996e84d.jpg)

I am testing it on XCode 10.2.1 on Simulator with iOS 12.2. There is a similar issue https://github.com/aws-amplify/aws-sdk-ios/issues/1082 but I guess I already have facebook and Userpool configured in configuration file. So it must me something else which is causing this issue.
Let me know if you need anymore details. Thanks!

**Comment by gorevoy** [comment]
I also have same issue. It because getDetails/getSession not coming to the block:

`[[self.user getDetails] continueWithBlock:^id _Nullable(AWSTask<AWSCognitoIdentityUserGetDetailsResponse *> * _Nonnull task) {
        // not coming to the block
}];`

I only see in log:

> Warning: A long-running operation is being executed on the main thread. 
>  Break on awsbf_warnBlockingOperationOnMainThread() to debug.

**Comment by royjit** [comment]
I was able to reproduce this issue when I do not have `AWSUserPoolsSignIn` in my podfile. Could you please verify that this is what is happening for you as well?

**Comment by vijayendra-tripathi** [comment]
@royjit @rohandubal 

Looks like things have changed in API since I last used it. AWS pods version were 2.9.9 back then and today when I checked Amplify docs, they are now 2.11.1. I updated pods and used latest code as recommended by them. But I still same error. However I get following warning on console - 

> Warning once only: UITableView was told to layout its visible cells and other contents without being in the view hierarchy (the table view or one of its superviews has not been added to a window). This may cause bugs by forcing views inside the table view to load and perform layout without accurate information (e.g. table view bounds, trait collection, layout margins, safe area insets, etc), and will also cause unnecessary performance overhead due to extra layout passes. Make a symbolic breakpoint at UITableViewAlertForLayoutOutsideViewHierarchy to catch this in the debugger and see what caused this to occur, so you can avoid this action altogether if possible, or defer it until the table view has been added to a window. Table view: <UITableView: 0x7fa2bf87e200; frame = (4 4; 271 142); clipsToBounds = YES; autoresize = RM+BM; gestureRecognizers = <NSArray: 0x600002f591a0>; layer = <CALayer: 0x6000021b4660>; contentOffset: {0, 0}; contentSize: {271, 0}; adjustedContentInset: {0, 0, 0, 0}; dataSource: (null)>
> 2019-10-06 18:39:07.270339+0530 logits-api-test[5982:95896] Task <5E70DBFE-D57E-4998-899F-FCEE368CE2EC>.<1> HTTP load failed, 0/0 bytes (error code: -999 [1:89])

I use following pods -
pod 'AWSUserPoolsSignIn', '~> 2.11.0' 
pod 'AWSAuthUI', '~> 2.11.0' 
pod 'AWSMobileClient', '~> 2.11.0' 


One more thing I noticed was that when I submit facebook token submission using following function -

```
AWSMobileClient.default().federatedSignIn(providerName: IdentityProvider.facebook.rawValue, token: "EAAF7......v8ZD") { (userState, error)  in
            if let error = error {
                print("Federated Sign In failed: \(error.localizedDescription)")
            }
            if let userState = userState {
                print("User state: \(userState)")
            }
        }
```
This prints userState as "SignedIn" but I don't see user being created in Cognito at AWS.

**Comment by royjit** [comment]
Hi @vijayendra-tripathi 

Sorry for the delayed response.
Were you able to fix the issue? I tried setting up myself and I had no problem in displaying the drop in UI. 
I called the UI like this:

```
let signInOptions = SignInUIOptions(canCancel: true)
AWSMobileClient.default().showSignIn(navigationController: self.navigationController!,
                                             signInUIOptions: signInOptions,
                                             { (userState, error) in
                                                
        })
```

If you want to have a user created in UserPool when you federate via a social provider, you have to use the [hostedUI](https://aws-amplify.github.io/docs/sdk/ios/authentication#using-hosted-ui-for-authentication) from Cognito.
